File: app/src/main/java/com/quandoo/androidtask/api/Reservation.java
package com.quandoo.androidtask.api;

import com.google.gson.annotations.SerializedName;

public class Reservation {

    @SerializedName("user_id")
    long userId;

    @SerializedName("table_id")
    long tableId;

    @SerializedName("id")
    long id;


    public Reservation(long userId, long tableId, long id) {
        this.userId = userId;
        this.tableId = tableId;
        this.id = id;
    }

    @Override
    public String toString() {
        return "Reservation{" +
                "userId=" + userId +
                ", tableId=" + tableId +
                ", id=" + id +
                '}';
    }

    public long getUserId() {
        return userId;
    }

    public long getTableId() {
        return tableId;
    }

    public long getId() {
        return id;
    }
}

File: app/src/main/java/com/quandoo/androidtask/data/models/Reservation.kt
package com.quandoo.androidtask.data.models

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Reservation(userId: Long, tableId: Long, id: Long) : Serializable {

    @SerializedName("user_id")
    var userId: Long = 0
        internal set

    @SerializedName("table_id")
    var tableId: Long = 0
        internal set

    @SerializedName("id")
    var id: Long = 0
        internal set


    init {
        this.userId = userId
        this.tableId = tableId
        this.id = id
    }

    override fun toString(): String {
        return "Reservation{" +
                "userId=" + userId +
                ", tableId=" + tableId +
                ", id=" + id +
                '}'.toString()
    }
}
